brainstormforce/astra
# AST-2355 - Added separators in visibility controls in customizer builder components

### Description
- Section dividers missing in customizer controls

### Screenshots
- https://d.pr/i/6YjOYr

### Types of changes
- Non-breaking change

### How has this been tested?
- Check any builder's element
- Check footer component with responsive hide settings

### Checklist:
